src/lib/Components/Bidding/Screens/RegistrationResult.tsx

```diff
@@ -38,9 +38,9 @@ const registrationCompleteMessage = {
 const registrationPendingMessage = {
   title: "Registration pending",
   description:
-    "You will receive an email from an Artsy specialist once your registration has been confirmed.\n" +
+    "Artsy is reviewing your registration and you will receive an email when it has been confirmed. Please email specialist@artsy.net with any questions.\n" +
     "\n" +
-    "In the meantime, you can still view works and watch lots that you’re interested in.",
+    "In the meantime, you can still view works and watch lots you’re interested in.\n",
```

Comment:
Is this new trailing new line for padding?

Reply:
+1 to this question? Is this newline addition intentional?